x-pack/plugins/monitoring/server/routes/api/v1/apm/metric_set_instance.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const metricSet = [
```

Comment:
I'm trying to understand what will happen with this object / array. Will this be used to build the UI?

Reply:
Yea this is the pattern monitoring has established as a way to detail what data will be sent down to show in the `Instances` view as well as the `Overview` view.